context/app/api/vitessce_confs/assay_confs.py

```diff
@@ -0,0 +1,207 @@
+import re
+
+from vitessce import (
+    VitessceConfig,
+    MultiImageWrapper,
+    OmeTiffWrapper,
+    DataType as dt,
+    FileType as ft,
+)
+
+from .utils import (
+    group_by_file_name,
+    get_matches,
+)
+from .base_confs import (
+    ImagingViewConf,
+    ScatterplotViewConf,
+    ImagePyramidViewConf,
+    SPRMViewConf,
+)
+from .constants import Assays, AssetPaths, CommonsTypeClient
+
+
+class SeqFISHViewConf(ImagingViewConf):
+    def build_vitessce_conf(self):
+        file_paths_found = [file["rel_path"] for file in self._entity["files"]]
+        full_seqfish_reqex = "/".join(
+            [
+                AssetPaths.IMAGE_PYRAMID_DIR.value,
+                AssetPaths.SEQFISH_HYB_CYCLE_REGEX.value,
+                AssetPaths.SEQFISH_FILE_REGEX.value,
+            ]
+        )
+        found_images = get_matches(file_paths_found, full_seqfish_reqex)
+        # Get all files grouped by PosN names.
+        images_by_pos = group_by_file_name(found_images)
+        confs = []
+        # Build up a conf for each Pos.
+        for images in images_by_pos:
+            image_wrappers = []
+            pos_name = self._get_pos_name(images[0])
+            vc = VitessceConfig(name=pos_name)
+            dataset = vc.add_dataset(name=pos_name)
+            for img_path in sorted(images, key=self._get_hybcycle):
+                img_url, offsets_url = self._get_img_and_offset_url(
+                    img_path, AssetPaths.IMAGE_PYRAMID_DIR.value
+                )
+                image_wrappers.append(
+                    OmeTiffWrapper(
+                        img_url=img_url,
+                        offsets_url=offsets_url,
+                        name=self._get_hybcycle(img_path),
+                    )
+                )
+            dataset = dataset.add_object(MultiImageWrapper(image_wrappers))
+            vc = self._setup_view_config_raster(vc, dataset)
+            conf = vc.to_dict()
+            # Don't want to render all layers
+            del conf["datasets"][0]["files"][0]["options"]["renderLayers"]
+            confs.append(conf)
+        self.conf = confs
+        return self
```

Comment:
Unless I'm missing it, the class just does one thing: build confs. I don't think you would chain methods like this? So I'd suggest just getting rid of `self.conf` and then:
```suggestion
        return confs
```

and in `client.py`:
```diff
- vc.build_vitessce_conf()
- return vc.conf
+ return vc.build_vitessce_conf()
```

Reply:
Done!